src/app/views/marketplace/marketplace-app.controller.coffee

```diff
@@ -169,6 +169,7 @@ angular.module 'mnoEnterpriseAngular'
           (response) ->
             vm.reviews.list.splice(key, 1)
             updateAverageRating(response.average_rating)
+            vm.reviews.pageChangedCb(vm.app.id, vm.reviews.nbItems, vm.reviews.page = 1)
```

Comment:
@alexnoox , using unshift or splice cause wrong workflow, as it does not team up with pagination at all:

-Being in any page that is not first, and adding a question or review, it will show as the first one on that page instead of in page 1.
-When deleting the last comment/question in pages > 1, it will show a message "no reviews/questions", when there are still in other pages. 

This is the only reasonable solution I found. Since a user, normally would post/delete once, it won't over hit the backend.

Thoughts?